src/kartikeyDJune20/CountElementsinPreDefindString.java

```diff
@@ -0,0 +1,37 @@
+package kartikeyDJune20;
+
+public class CountElementsinPreDefindString {
+	static void countInString(String input) {
+		int cntDig = 0;
+		int cntUpper = 0;
+		int cntLower = 0;
+		int cntSpecial = 0;
+		for (int index = 0; index < input.length(); index++) {
+
+			char ch = input.charAt(index);
+			if (Character.isDigit(ch))
+				cntDig++;
+			else if (Character.isLetter(ch)) {
+
+					if (Character.isLowerCase(ch))
+					cntLower++;
```

Comment:
else if (Character.isLetter(ch)) without this block you are able to achieve the output.

Reply:
No need to check for letters, because it will already going to cover under uppercase and lowercase.